frontend/src/theme/mainTheme/mainColors.ts

```diff
@@ -0,0 +1,40 @@
+import { transparentize } from "color2k"
+import { colors } from "../primitives/colors"
+
+const genericColors = {
+  ...colors,
+  bodyText: colors.gray90,
+  danger: "#9d292d",
+  info: "#1e6777",
+  success: "#176c36",
+  warning: "#947c2d",
+  primary: "#f63366",
+  secondary: colors.gray60,
+  lightestGray: colors.gray20,
+  lightGray: colors.gray30,
+  gray: colors.gray60,
+  darkGray: colors.gray70,
+  red: colors.red80,
+  blue: colors.blue80,
+}
+
+export default {
+  ...genericColors,
+  // Alerts
+  alertErrorBorderColor: transparentize(genericColors.red, 0.8),
+  alertErrorBackgroundColor: transparentize(genericColors.red, 0.8),
+  alertErrorTextColor: genericColors.danger,
+  alertInfoBorderColor: transparentize(colors.blue80, 0.9),
+  alertInfoBackgroundColor: transparentize(colors.blue80, 0.9),
+  alertInfoTextColor: genericColors.info,
+  alertSuccessBorderColor: transparentize(colors.green80, 0.8),
+  alertSuccessBackgroundColor: transparentize(colors.green80, 0.8),
```

Comment:
can we define/add to genericColors and replace here? 
`green: colors.green80`

It looks like green80 is used for other stuff besides alert (`$code-color` and `$kbd-color`)